src/core/components/Paginate/Paginate.scss

```diff
@@ -11,16 +11,21 @@
   justify-content: space-between;
 }
 
+.Paginate {
+  padding: 0 10%;
```

Comment:
If possible I think an explicit pixel value would be better; for wider screens, especially on mobile, it will allow the buttons to reach closer to the edge of the screen which is better for usability.